content/docs/reference/pkg/alicloud/actiontrail/gettrails.md

```diff
@@ -291,26 +291,26 @@ The following output properties are available:
 
     <dt class="property-"
             title="">
-        <span>name<wbr>Regex</span>
+        <span>names</span>
         <span class="property-indicator"></span>
-        <span class="property-type">string?</span>
+        <span class="property-type">string[]</span>
     </dt>
-    <dd>{{% md %}}{{% /md %}}</dd>
+    <dd>{{% md %}}A list of trail names.
+{{% /md %}}</dd>
 
     <dt class="property-"
             title="">
-        <span>names</span>
+        <span>name<wbr>Regex</span>
         <span class="property-indicator"></span>
-        <span class="property-type">string[]</span>
+        <span class="property-type">string</span>
```

Comment:
Is this change right?

Reply:
Ah, maybe it's just that the diff rendering is making this look weird.

Reply:
Just adding a comment for others wondering why this diff is ok to ignore. This is because the properties are sorted by whether or not they are required. It's from https://github.com/pulumi/pulumi/pull/4351.